views/_header.ejs

```diff
@@ -1,29 +1,59 @@
-<header>
-    Here comes the header!!!!
-</header>
+<link rel="stylesheet" href="css\header.css">
+<nav id="mainnavbar" class="navbar navbar-expand-lg navbar-light ">
+    <a class="navbar-brand" href="#"></a>
+  <button onclick="myFunction()" id="menubutton" class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarToggleExternalContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+  <div class="collapse navbar-collapse" id="navbarToggleExternalContent">
+    <ul class="navbar-nav ml-auto">
+      <li class="nav-item">
+        <a class="nav-link" href="/about">ABOUT</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="/events">EVENTS</a>
+      </li>
+      <!-- <li class="nav-item dropdown">
+        <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+          Dropdown
+        </a>
+        <div class="dropdown-menu" aria-labelledby="navbarDropdown">
+          <a class="dropdown-item" href="#">Action</a>
+          <a class="dropdown-item" href="#">Another action</a>
+          <div class="dropdown-divider"></div>
+          <a class="dropdown-item" href="#">Something else here</a>
+        </div>
+      </li> -->
+      <li class="nav-item">
+        <a class="nav-link " href="/membership">MEMBERSHIP</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link " href="/alumni">ALUMNI</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link " href="/blog">BLOG</a>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link " href="/contact">CONTACT</a>
+      </li>
+    </ul>
+  </div>
+</nav>
 
-<ul style="list-style-type:none;">
-    <li>
-        <a href="/">Home</a>
-    </li>
-    <li>
-        <a href="/about">About</a>
-    </li>
-    <li>
-        <a href="/alumni">Alumni</a>
-    </li>
-    <li>
-        <a href="/blog">Blog</a>
-    </li>
-    <li>
-        <a href="/events">Events</a>
-    </li>
-    <li>
-        <a href="/membership">Membership</a>
-    </li>
-    <li>
-        <a href="/contact">Contact</a>
-    </li>
-
-
-</ul>
\ No newline at end of file
+<script>
```

Comment:
1.Make a new folder in assets and add scripts, for ex. assets/scripts/header.js. 
2.Try giving meaningful name to functions rather than myFunction.

Reply:
Made a new folder name scripts in the assests folder and added header.js file in that. Changed function name to the more useful one. Thanks for your important guidance.